terraform/api_gateway.tf.json.template.py

```diff
@@ -73,7 +74,7 @@ def for_domain(cls, domain):
             "aws_route53_zone": {
                 zone.slug: {
                     "name": zone.name,
-                    "private_zone": False
+                    "private_zone": False,
```

Comment:
```suggestion
                    "private_zone": False
```

No trailing commas. See contribution guide.